src/app/common/transition.ts

```diff
@@ -0,0 +1 @@
+export const transition = 'all 0.2s cubic-bezier(0.23, 1, 0.32, 1)';
```

Comment:
Use panda for this https://panda-css.com/docs/theming/tokens#easings

Reply:
Ok, thx, will relocate.